src/Models/FillsItemModel.vala

```diff
@@ -20,7 +20,7 @@
 * Authored by: Alessandro "alecaddd" Castellani <castellani.ale@gmail.com>
 */
 
-public class Akira.Models.FillsItemModel : GLib.Object {
+public class Akira.Models.FillsItemModel : Models.ItemModel {
```

Comment:
The `blending_mode` property should have been declared here, since it's a specific property of this particular model